### PR TITLE
KKP Docs introduction changes (2.20 and master)

### PR DESCRIPTION
**Pull request by Mita-Kubermatic**
Changed the introduction to include all providers

### CODE REVIEW

File: content/kubermatic/master/_index.en.md [hunk 1]
```diff
@@ -12,7 +12,10 @@ date = 2019-04-27T16:06:34+02:00
...
 
 
-KKP is directly integrated with leading cloud providers such as [Amazon Web Services](https://docs.kubermatic.com/kubermatic/master/architecture/requirements/support_policy/provider_support_matrix/aws/aws/), [Azure](https://docs.kubermatic.com/kubermatic/master/architecture/requirements/support_policy/provider_support_matrix/azure/azure/), DigitalOcean, [Google Compute Engine](https://docs.kubermatic.com/kubermatic/master/architecture/requirements/support_policy/provider_support_matrix/google_cloud/gcp/), Hetzner, OpenStack, Packet and [VMware vSphere](https://docs.kubermatic.com/kubermatic/master/architecture/requirements/support_policy/provider_support_matrix/vsphere/vsphere/) as well as any provider offering Ubuntu 16.04 or greater, even in your own datacenter.
+KKP is directly integrated with leading cloud providers including Amazon Web Services (AWS), Google Cloud, Azure, Openstack, VMware vSphere, Open Telekom Cloud, Digital Ocean, Heztner, Alibaba Cloud, kubeadm, Equinix Metal, Nutanix and ARM.
```
Comment (embik) [inline content/kubermatic/master/_index.en.md:15]: small typo: `Heztner` -> `Hetzner`.

In addition, "ARM" and "kubeadm" or no cloud providers, I think they shouldn't be listed the same way as the others (ARM is a CPU architecture, and kubeadm is a tool to bootstrap/join a Kubernetes cluster). IMHO it still makes sense to mention `kubeadm` here, just in a different context (e.g. "with leading cloud provider including X, Y and Z, and allows joining Kubernetes nodes from any infrastructure with `kubeadm`", something like that).

**Review by embik — CHANGES_REQUESTED** [review]

File: content/kubermatic/master/_index.en.md (hunk 1 above)
Comment (Mita-Kubermatic) [reply content/kubermatic/master/_index.en.md:15]: We can remove kubeadm and ARM from the list and add 2 separate sentences for each. Can you suggest the change?

**Comment by embik** [comment]
@Mita-Kubermatic I'm afraid the PR needs to be rebased due to a conflict on the `master` branch.

**Review by embik — APPROVED** [review]
/approve

**Pull request merged**